the-craft-command/introduction.md

```diff
@@ -127,7 +127,7 @@ $ python craft validator LoginValidator
 Be sure to read the [Validation](../advanced/validation.md) documentation to learn more about validators.
 {% endhint %}
 
-### Creating Model Definition Docstrings
+<!-- ### Creating Model Definition Docstrings
```

Comment:
Here it's not possible anymore with the ORM (at least I haven't found it) so I just comment out this part.
We could uncomment this when this feature will be re-added